python/py-mysqlclient/Portfile

```diff
@@ -39,32 +39,50 @@ if {${name} ne ${subport}} {
             ![variant_isset mysql8] &&
             ![variant_isset mariadb55] &&
             ![variant_isset mariadb10_2] &&
+            ![variant_isset mariadb10_3] &&
+            ![variant_isset mariadb10_4] &&
+            ![variant_isset mariadb10_5] &&
             ![variant_isset percona55]} {
-            return -code error "you must select either mysql5, mysql8, mariadb55, mariadb10_2 or percona55"
+            return -code error "you must select either mysql5, mysql8, mariadb55, mariadb10_2 through mariadb10_5 or percona55"
```

Comment:
Could you please correct the typo, as well.  It should say mysql57 rather than mysql5.